Shared/Samples/Change map view background/README.md

```diff
@@ -0,0 +1,36 @@
+# Change map view background
+
+Customize map view's background by changing its grid properties.
+
+| ChangeMapViewBackground | Settings |
+| ----------------- | ------------- |
+|![Image of change map view background](change-map-view-background.png)|![Image of change map view background](change-map-view-background-settings.png)|
```

Comment:
```suggestion
![Image of change map view background](change-map-view-background-settings.png)|
```
I think it is enough to have only 1 screenshot for this sample, that shows the various controls.